frame/support/src/weights.rs

```diff
@@ -315,6 +353,11 @@ impl sp_runtime::traits::Printable for PostDispatchInfo {
 		match self.actual_weight {
 			Some(weight) => weight.print(),
 			None => "max-weight".print(),
+		};
+		" pays_fee=".print();
```

Comment:
```suggestion
		"pays_fee=".print();
```

Every `print` call will be put into a new line